applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -345,9 +345,9 @@ To perform DFU using the `nRF Connect Device Manager`_ mobile app, complete the
 
          a. Ensure that you can access the :file:`root.suit` envelope from your phone or tablet.
          #. In the mobile app, scan and select the device to update.
-         #. Switch to the :guilabel:`Image` tab and tap on :guilabel:`ADVANCED` in the upper right corner of the app.
-         #. In the **Firmware Upload** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope.
-         #. Tap the :guilabel:`UPLOAD` button.
+         #. Switch to the :guilabel:`Image` tab.
+         #. In the **Firmware Upgrade** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope.
```

Comment:
You may also use the `.zip` file. It covers a little bit more scenarios :slightly_smiling_face: